cmdk/src/index.tsx

```diff
@@ -592,10 +593,10 @@ const Item = React.forwardRef<HTMLDivElement, ItemProps>((props, forwardedRef) =
     props.forceMount
       ? true
       : context.filter() === false
-      ? true
-      : !state.search
-      ? true
-      : state.filtered.items.get(id) > 0,
+        ? true
+        : !state.search
+          ? true
+          : state.filtered.items.get(id) > 0,
```

Comment:
Couple of prettier issues here, if you wouldn't mind fixing :)

Reply:
Done!